form.tex

```diff
@@ -0,0 +1,92 @@
+\documentclass[a4paper]{article}
+
+\usepackage[bmargin=3cm, tmargin=2cm]{geometry}
+\usepackage{graphicx}
+\usepackage{xspace}
+\usepackage[latin1]{inputenc}
+\usepackage[pdftex]{hyperref}
+\setlength\parindent{0pt}
+
+\usepackage{helvet}
+\renewcommand*\familydefault{\sfdefault} %% Only if the base font of the document is to be sans serif
+\usepackage[T1]{fontenc}
+
+\pagenumbering{gobble}
+
+%%%% Defines for form filling
+\newcommand{\signature}{%
+    Signature:\\
+    \vspace{1cm}\\
+    \rule{7cm}{1pt}\\
+    Date: \vbox{\hbox{\TextField[width=6cm]{\null}}\kern2pt\hrule}
+}
+
+\begin{document}
+
+\newcommand{\compyear}{2018\xspace}
+\newcommand{\competition}{SourceBots \compyear}
+
+\begin{minipage}[c]{0.30\textwidth}
+\includegraphics[height=2cm]{fig-SourceBots}
+\end{minipage}
+\hspace{0.03\textwidth}
+\begin{minipage}[c]{0.67\textwidth}
+\begin{center}
+{\large \textbf{\competition Media Consent Release Form} \par}
+\end{center}
+\end{minipage}
+\vspace{1cm}
+
+\textbf{This is not a ticket.}\\\\ However, this form is required for entry into the \competition competition.\\
+\vspace{0.3cm}
+
+\textbf{\large Media Consent}\\
+
+By signing this release form, you (the individual mentioned below) consent to your image, voice, biographical material, or other information being recorded by:
+\begin{itemize}
+    \item The University of Southampton
+    \item SourceBots
+    \item Current or future sponsors of SourceBots
+\end{itemize}
+Your recordings may be used in publications (including but not limited to; press releases, websites, blog posts, livestreams, videos, brochures, and posters) for the purpose of publicising/promoting the event and future events held by the above parties.
+
+\vspace{0.5cm}
+\textbf{\large Privacy Policy}\\
+
+Once signed, this form will be stored securely and encrypted at the University of Southampton for the entire time any recordings made at \competition are used. All recordings will be openly available to the internet. You have the right to request for the deletion of any personal data (including images) as per the EU General Data Protection Regulation (GDPR).
```

Comment:
I've been led to believe that a photo contains someone's personal information as long as their face is recognisable in it, not just if they're the subject of the photo.

Therefore I don't think tagging the photo with someone's name changes the situation, as there still exists personal information which must be removed if they ask. In that situation, it would be a simple case of scrolling through 500-odd photos, looking for someone's face in them, which humans are pretty fast at doing.